airflow/utils/log/es_task_handler.py

```diff
@@ -68,6 +100,76 @@ def __init__(self, base_log_folder, filename_template,
         self.mark_end_on_close = True
         self.end_of_log_mark = end_of_log_mark
 
+    def set_context(self, ti):
+        if self.write_stdout == True:
+            self.writer = parentStdout()
+            sys.stdout = self.writer
+            self.taskInstance = self._process_taskInstance(ti)
+            self.handler = logging.StreamHandler(stream=sys.stdout)
+            self.handler.setFormatter(JsonFormatter(self.taskInstance))
+            self.handler.setLevel(self.level)
+        elif self.write_stdout == False:
+            super(ElasticsearchTaskHandler, self).set_context(ti)
+        self.mark_end_on_close = not ti.raw
+
+    def emit(self, record):
+        if self.write_stdout == True:
+            self.formatter.format(record)
+            if self.handler is not None:
+                self.handler.emit(record)
+        elif not self.write_stdout:
+            super(ElasticsearchTaskHandler).emit(record)
+
+    def flush(self):
+        if self.handler is not None:
+            self.handler.flush()
+
+    def _process_taskInstance(self, ti):
```

Comment:
@rhwang10 should this be `_process_task_instance` to stay consistently snake cased?